src/chains/ethereum_1/ethTransaction.ts

```diff
@@ -99,18 +99,17 @@ export class EthereumTransaction implements Transaction {
     if (!this._actionHelper) {
       throwNewError('Transaction serialization failure. Transaction has no actions.')
     }
-    const { chain, hardfork, nonce } = this._options
+    const { chainSettings } = this._chainState
+    const { chainForkType } = chainSettings
+    const { chainName, hardFork } = chainForkType
+    const trxOptions = { chain: chainName, hardfork: hardFork }
```

Comment:
what if chainName or hardFork is null?
I assume you want to check that both of those values are not null before adding to trxOptions - not sure why you removed this below in the code. If they are required values, why is chainForkType optional for options?